### PR TITLE
CLOUD-318 fix psmdb docker images build

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: cloud/jenkins/psmdb-docker.yml [hunk 1]
```diff
@@ -29,3 +29,11 @@
         name: GIT_BRANCH
         default: master
         description: Tag/Branch for percona/percona-server-mongodb-operator repository
+    - string:
+        name: GIT_PD_BRANCH
```
Comment (delgod) [inline cloud/jenkins/psmdb-docker.yml]:
```suggestion
        name: GIT_PD_REPO
```

File: cloud/jenkins/psmdb_docker_build.groovy [hunk 2]
```diff
@@ -1,8 +1,8 @@
 void build(String IMAGE_PREFIX){
     sh """
         cd ./source/
-        DOCKER_FILE_PREFIX=\$(echo ${IMAGE_PREFIX} | tr -d '.')
-        docker build --no-cache --squash -t perconalab/percona-server-mongodb-operator:master-${IMAGE_PREFIX} -f build/Dockerfile.\$DOCKER_FILE_PREFIX .
+        DOCKER_FILE_PREFIX=\$(echo ${IMAGE_PREFIX} | tr -d '.' | tr -d 'mongod')
+        docker build --no-cache --squash -t perconalab/percona-server-mongodb-operator:master-${IMAGE_PREFIX} -f percona-server-mongodb.\$DOCKER_FILE_PREFIX/Dockerfile percona-server-mongodb.\$DOCKER_FILE_PREFIX
```
Comment (delgod) [inline cloud/jenkins/psmdb_docker_build.groovy]:
```suggestion
        docker build --no-cache --squash -t perconalab/percona-server-mongodb-operator:master-${IMAGE_PREFIX} -f percona-server-mongodb.\$DOCKER_FILE_PREFIX/Dockerfile.k8s percona-server-mongodb.\$DOCKER_FILE_PREFIX
```

File: cloud/jenkins/psmdb-docker.yml (hunk 1 above)
Comment (hors) [reply cloud/jenkins/psmdb-docker.yml]: fixed

File: cloud/jenkins/psmdb_docker_build.groovy (hunk 2 above)
Comment (hors) [reply cloud/jenkins/psmdb_docker_build.groovy]: fixed

**Pull request merged**